src/NetEscapades.AspNetCore.SecurityHeaders/HeaderPolicyCollectionExtensions.cs

```diff
@@ -51,6 +51,9 @@ public static HeaderPolicyCollection AddDefaultSecurityHeaders(this HeaderPolicy
             builder.AddFrameAncestors().None();
         });
         policies.AddCrossOriginOpenerPolicy(x => x.SameOrigin());
+        policies.AddCrossOriginEmbedderPolicy(builder => builder.RequireCorp());
```

Comment:
I'm a _little_ nervous about adding this one, as this will cause people who have already set up CORS to suddenly have broken sites 😬 Unless the API also sets the `Cross-Origin-Resource_policy` header, the APIs won't work as I understand it...

Reply:
yes, I have this problem when people link streams and other sites. I remove this from the defaults.

Reply:
I always use this :) removed